mar/tiebout-action.hoon

```diff
@@ -0,0 +1,40 @@
+::
+::::  /hoon/actions/collection/mar
+  ::
+/?  309
+/-  hall
+/+  tiebout
+=,  tiebout
+=,  format
+::
+|_  act=action:tiebout
+::
+++  grow
+  |%
+  ++  tank  >act<
+  --
+::
+++  grab
+  |%
+  ++  noun  action:tiebout
+  ++  json
+    |=  jon=^json
+    ^-  action:tiebout
+    %-  (hard action:tiebout)
+    =<  (action jon)
+    |%
+    ++  action
```

Comment:
`|^` exists and might be nicer here.

Reply:
Nest-failed in a weird way, skipping it.

Reply:
Happy to help you figure it out if you want to, but I won't press you on it.